libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -894,7 +894,6 @@ public void updateCurrentLocationDrawable(@DrawableRes int gpsDrawable) {
       if (options.gpsDrawable() != gpsDrawable) {
         LocationComponentOptions newOptions = options.toBuilder()
           .gpsDrawable(gpsDrawable)
-          .padding(mapboxMap.getPadding())
```

Comment:
@Guardiola31337 - thanks for your reminder about this unused method. I also remove the padding here.
then this PR should be good to go.